CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# CLIMADA Contribution Guide
+
+Thank you for contributing to CLIMADA!
+
+## Overview
+
+Before you start, please have a look at our [Developer Guide][devguide].
+
+To contribute follow these steps:
+
+1. Fork the project on GitHub.
```

Comment:
I think we have to be a bit careful here. So far, all developer's directly affiliated with the WCR have been working directly on the repository (to simplify things as described in the Developer Guide). Thus, forking would only be for unrelated developer's. So, I am not sure whether this general formulation will not create more confusion.

Reply:
I think the [documentation](https://climada-python.readthedocs.io/en/stable/guide/Guide_Git_Development.html#Installing-CLIMADA-for-development) on devs environment and branching is already quite extensive. I felt like the instructions here were somehow outdated, but I simply copied them over from the `README.md` for consistency. I think it would suffice to only link to the documentation and drop the instructions here completely. Would you agree? (This also applies to your second comment)

Reply:
Yes, I would agree. Probably we should take one moment to step back and check whether the CLIMADA landing page is then still clear enough for newcommers.

Reply:
Fine by me. How do you want to proceed, then?

Reply:
I re-wrote the instructions here: https://github.com/CLIMADA-project/climada_python/pull/518/commits/6a991e746d76bc76ec3b89888a3480fa980222a6 Would you take another look?

Reply:
Amazing! Thanks, it is much improved now.